case_studies/galaxy_clustering/data_generation/background_prior.py

```diff
@@ -0,0 +1,305 @@
+import os
+import random
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from astropy.io import fits
+from scipy.stats import gennorm
+
+DES_DIR = Path(
+    "/nfs/turbo/lsa-regier/scratch/gapatron/desdr-server.ncsa.illinois.edu/despublic/dr2_tiles/"
+)
+DES_SUBDIRS = os.listdir(DES_DIR)
+
+
+class BackgroundPrior:
+    def __init__(self, image_size=4800):
+        super().__init__()
+        self.width = image_size
+        self.height = image_size
+        self.center_offset = (self.width / 2) - 0.5
+        self.bands = ["G", "R", "I", "Z", "Y"]
+        self.n_bands = 5
+        self.reference_band = 1
+        self.ra_cen = 50.64516228577292
+        self.dec_cen = -40.228830895890404
+        self.pixels_per_mpc = 80
+        self.mean_sources = 0.004
+        self.G1_beta = 0.6
+        self.G1_loc = 0
+        self.G1_scale = 0.035
+        self.G2_beta = 0.6
+        self.G2_loc = 0
+        self.G2_scale = 0.032
+        self.pixel_scale = 0.26
```

Comment:
pixel scale for coadds is 0.263, not 0.26, according to https://arxiv.org/pdf/2101.05765

Reply:
Just fixed this, and the speckled pattern!